app/models/application_choice.rb

```diff
@@ -185,9 +178,40 @@ def withdrawn_at_candidates_request?
     )
   end
 
+  def configure_initial_course_choice!(course_option)
+    update!(
+      course_option: course_option,
+      current_course_option: course_option,
+    )
+    update_course_option!(course_option)
```

Comment:
We are updating the record twice here, generating two audits which might be confusing, I wonder if we can avoid that?

Reply:
I thought this didn't matter because it's before applications are sent to a provider, but I think I have found a way to produce one audit (via `other_fields`).